README.md

```diff
@@ -6,7 +6,8 @@ Formats CMake-specific files to the desired format using [`cmake-format`](https:
 
 ## Usage
 
-To use this action, pass arguments to the `args` element as you would to `cmake-format` - these arguments will be used to format each CMake file. For example
+To use this action, pass arguments to the `args` element as you would to `cmake-format` - these arguments will be used to format each CMake file. To select the files which should be processed use the `find_regex` argument.
```

Comment:
How about we change 
```md
To select the files which should be processed use the `find_regex` argument.
```
to
```
For more fine-grained control over which files to process, use the `find_regex` argument; see below for an example.
```

Reply:
added in https://github.com/puneetmatharu/cmake-format-lint-action/pull/9/commits/c4c94b0b769e7fdc2430fcb79fdf2ff5e2d663b6